docs/install.md

```diff
@@ -83,6 +83,20 @@ within a temporary runtime directory.
 ### Ubuntu 14.04:
 You can try to carry on the work done here: <https://gist.github.com/dagelf/99ee07f5638b346adb8c058ab3d57492>
 
+### openSUSE Tumbleweed:
+
+> If you're running transactional server, first chroot into snapshot by
+> issuing `$ transactional-update shell`. It's also recommended that you don't put bees executables into read-write area if you did. 
```

Comment:
Trailing whitespace.  Also why is this formatted as a quote?

Reply:
I used WYSIWYG markdown editor so I didn't aware of trailing whitespace.
Quote it because it only applies to those running transactional server, like a optional step for specific situation.
You may change the formatting freely, it's just a sample.